Eatery/Controllers/Eateries/Campus/CampusEateriesViewController.swift

```diff
@@ -279,3 +291,52 @@ extension CampusEateriesViewController: UISearchControllerDelegate {
     }
 
 }
+
+
+extension CampusEateriesViewController: UINavigationControllerDelegate {
+    func navigationController(_ navigationController: UINavigationController, willShow viewController: UIViewController, animated: Bool) {
+        viewController.extendedLayoutIncludesOpaqueBars = true
+
+        let isParallax = viewController is ImageParallaxScrollViewController
+        navigationController.setNavigationBarHidden(isParallax, animated: true)
+    }
+}
+
+
+
+
+/*  The code below was taken from EateriesSharedViewController and it is unclear if it is needed
+    
+    extension CampusEate
```

Comment:
I think it'd be good to move the location manager into `CampusEateriesVC`

Reply:
yup -- i believe location is still needed to display how far away an eatery is